gnupg/_parsers.py

```diff
@@ -1027,6 +1033,9 @@ def sub(self, args):
         subkey = [args[4], args[11]]
         self.curkey['subkeys'].append(subkey)
 
+    def rvk(self, args):
+        self.curuid == None
```

Comment:
I don't remember why I reset `curuid`, but that fixed the problem at the time. I'm trying to get back and understand what I did and why.